server/migrations/1680744387850_order.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+    pgm.createTable(`orders`,{
+        order_id: "id",
+        table_id: {type: 'integer', notNull:"true", references:'user_tables', onDelete: 'cascade'},
```

Comment:
em hiểu `onDelete: 'cascade'` là gì ko?

Reply:
dạ thì nếu xóa cái bàng có cái table_id đó thì sẽ xóa toàn bộ dữ liệu liên quan tới cái table_id đó